fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -54,6 +54,9 @@ def self.start(user: nil, is_swift_fastfile: false)
       spinner.auto_spin
 
       ios_projects = Dir["**/*.xcodeproj"] + Dir["**/*.xcworkspace"]
+      ios_projects.delete_if do |path|
+        Gem.path.any? { |gem_path| File.expand_path(__dir__, path).start_with?(gem_path) }
```

Comment:
@chiastolite 

[Q] Where do you want to refer with `__dir__`?

I think this should be `Dir.pwd`. I'd appreciate if you could check this again and make sure this works as you intended.

Reply:
[IMO] `File.expand_path(path)` would be your solution 👍

Reply:
> [IMO] File.expand_path(path) would be your solution 👍

Oh, you’re right! I will fix it.